server/v2/stf/stf.go

```diff
@@ -391,6 +391,7 @@ type executionContext struct {
 	meter  gas.Meter
 	events []event.Event
 	sender []transaction.Identity
+	Cache  ModuleContainer
```

Comment:
will this be a source of concurrency issues, when parrallized txs become a thing?

Reply:
do you mean by using the same ctx for many txs?